src/01-dinosaur-facts.js

```diff
@@ -44,7 +69,18 @@ function getLongestDinosaur(dinosaurs) {}
  *  getDinosaurDescription(dinosaurs, "incorrect-id");
  *  //> "A dinosaur with an ID of 'incorrect-id' cannot be found."
  */
-function getDinosaurDescription(dinosaurs, id) {}
+function getDinosaurDescription(dinosaurs, id) {
+  // assign foundDino variable to the result of dinosaus.find() by id
+  const foundDino = dinosaurs.find(dino => dino.dinosaurId === id) 
+  if (foundDino) {
+    // return description of dinosaur
```

Comment:
remember with string interpolation we can make this a multi-line string.  

this is also a good spot for a helper function to abstract out all of the string manipulation